packages/astro/src/internal/index.ts

```diff
@@ -77,11 +76,11 @@ export interface AstroComponentFactory {
   isAstroComponentFactory?: boolean;
 }
 
-export const createComponent = (cb: AstroComponentFactory) => {
+export function createComponent(cb: AstroComponentFactory) {
```

Comment:
Minor code style nit: I think functions are generally better because they all get hoisted and don’t depend on order of definition like `const` does